### ISSUE TITLE
embulk-output-mysql: This plugin doesn't support json type.

### ISSUE THREAD

**Issue by adk9** [embulk/embulk-output-jdbc#150]
According to the documentation, `json` is one of the supported `value_type` for columns. However, when trying to import JSON input to the MySQL output, I run into the following error:
```
Error: java.lang.UnsupportedOperationException: This plugin doesn't support json type. Please try to upgrade version of the plugin using 'embulk gem update' command. If the latest version still doesn't support json type, please contact plugin developers, or change configuration of input plugin not to use json type. 
```

Looking at the code, only the PostgreSQL output plugin seems to implement the `jsonColumn` method. Is this intentional?

**Comment by hito4t** [comment]
Hi
embulk-output-jdbc plugins basically support json.
But `replace` mode hasn't support json yet.
(`replace` mode will create output table from input schema, but cannot create column of the table from json column in the input schema.)

> Looking at the code, only the PostgreSQL output plugin seems to implement the jsonColumn method. Is this intentional?

Because PostgreSQL has `json` type.
Other RDMSs don't have `json` type, so the `jsonColumn` metho was not implemented.

The following PR will create `CLOB` column for json column in input, same as string column in input.
https://github.com/embulk/embulk-output-jdbc/issues/150
I think this PR will solve this issue.

**Comment by hito4t** [comment]
I've merged #150 and released!

**Comment by jdotjdot** [comment]
Yay!
